frontend/src/components/Leaderboard.tsx

```
@@ -38,7 +64,7 @@ export default function Leaderboard() {
                     <p className={styles.emphasisText}>Want to compete?</p>
                     <p>Top 5 win prizes</p>
                 </div>
-                <button onClick={() => setShowLeaderboard(true)}>
+                <button onClick={() => updateLeaderboardPreference(true)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)